spec/directions/bake_chapter_key_equations_spec.rb

```diff
@@ -40,11 +45,14 @@
 
   context 'when append_to is nil' do
     it 'works' do
+      described_class.v1(chapter: chapter, metadata_source: metadata_element)
       expect(
-        described_class.v1(chapter: chapter, metadata_source: metadata_element)
+        chapter
       ).to match_normalized_html(
         <<~HTML
           <div data-type="chapter">
+            <div data-type="page">
+          </div>
```

Comment:
Intentional to have page closing tag on next line unindented? Reads weirdly.

Reply:
the spec fails unless it's formatted this way unfortunately